src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -375,6 +374,9 @@ protected RoslynDefinitionBucket GetOrCreateDefinitionBucket(DefinitionItem defi
                 }
             }
 
+            public sealed override Task ReportMessageAsync(string message)
+                => throw new InvalidOperationException("This should never be called in the streaming case.");
+
             public sealed override Task ReportProgressAsync(int current, int maximum)
             {
                 // https://devdiv.visualstudio.com/web/wi.aspx?pcguid=011b8bdf-6d56-4f87-be0d-0092136884d9&id=359162
```

Comment:
Not related to you but an #if false always draws attention: this was closed ages ago.... :smile: